launchCronAndLogToMXDev.sh

```diff
@@ -0,0 +1,12 @@
+
+MXDEV_URL=$(cat ~/MXControl/config/MXDEV_URL)
```

Comment:
load in the profile rather than doing this, tbh

Reply:
no mxprofile in MXUtil. We could change that, but yeah.